src/main/java/com/google/devtools/build/lib/bazel/bzlmod/BazelLockFileModule.java

```diff
@@ -117,6 +117,17 @@ public void afterCommand() {
     // lockfile that are still up-to-date and adding the newly resolved extension results.
     BazelLockFileValue newLockfile =
         BazelLockFileValue.builder()
+            // HACK: Flipping the flag `--incompatible_use_plus_in_repo_names` causes the canonical
```

Comment:
Couldn't this get us in trouble if we have to bump the version for some other reason? We could stick to even increments for as long as the flag exists to avoid that.

Reply:
indeed! I added a mention to even increments in the comment.